tests/Sdk/Acceptance/Presentation/RestApi/Controller/v1/RunTaskControllerCest.php

```diff
@@ -34,15 +36,51 @@ class RunTaskControllerCest
      *
      * @return void
      */
-    public function iSeeJsonResponseAfterCallHelloWorldEndpoint(AcceptanceTester $I): void
+    public function iSeeJsonResponseAfterCallRunTaskEndpoint(AcceptanceTester $I): void
     {
-        $I->sendPost(static::ENDPOINT, ['world' => 'World', 'somebody' => 'World']);
+        $I->haveHttpHeader('Content-Type', 'application/json');
+        $I->sendPost(static::ENDPOINT, [
+            OpenApiField::DATA => [
```

Comment:
I believe, it can be extracted to the separate Helper

Reply:
Fixed